src/plugins/intel_gpu/include/intel_gpu/runtime/memory.hpp

```diff
@@ -110,7 +110,7 @@ struct memory {
 
     virtual event::ptr copy_to(stream& stream, memory& other, bool blocking = true) const {
         const auto zero_offset = 0;
-        const auto data_size = other._bytes_count;
+        const auto data_size = _bytes_count;
```

Comment:
Do we have a check that dst size is big enough somewhere? If no, I think it should be added